multinode-demo/common.sh

```diff
@@ -47,3 +47,22 @@ export RUST_BACKTRACE=1
 [[ $(uname) = Linux ]] && (set -x; sudo sysctl -w net.core.rmem_max=26214400 1>/dev/null 2>/dev/null)
 
 SOLANA_CONFIG_DIR=${SNAP_DATA:-$PWD}/config
+
+rsync_url() { # adds the 'rsync://` prefix to URLs that need it
+  local url="$1"
+
+  if [[ "$url" =~ ^.*:.*$ ]]; then
```

Comment:
can do this more simply, if you like: ${url} != ${url%:*}
```[rwalker@anodyne .ssh]$ hi=foothere
[rwalker@anodyne .ssh]$ [[ $hi != ${hi%:*} ]] && echo colon!
[rwalker@anodyne .ssh]$ hi=foo:there
[rwalker@anodyne .ssh]$ [[ $hi != ${hi%:*} ]] && echo colon!
colon!
```

Reply:
I think I prefer the existing form so imma keep it as is